data/src/buffer.rs

```diff
@@ -60,19 +60,36 @@ impl From<config::Buffer> for Settings {
 }
 
 #[derive(Debug, Clone, Copy, Default, Deserialize)]
-pub enum InputVisibility {
+#[serde(rename_all = "kebab-case")]
+pub struct TextInput {
```

Comment:
Remove `kebab-case` since we're going snake_case. We only want `kebab-case` on enum variants